docker/labkey-20.3-community/README.md

```diff
@@ -0,0 +1,48 @@
+samples/docker/labkey-20.3-community
+==========
+
+This folder contains a sample docker-compose file and sample Docker configurations to stand up an instance of LabKey Server Community edition for testing.
+
+### IMPORTANT: Not Recommended for Production use
+ This image was meant for trying out LabKey Server and not meant for running a production server or for storing important biomedical data. 
+
+#### Technology configuration:
+* LabKey Server 20.3.2 
+* Tomcat 9
+* OpenJDK-14
+* PostgreSQL 11
+
+
+## Usage 
+
+### Recommended configuration steps
+
+1. Clone this repo
+1. Edit the labkey/config/labkey.xml and choose your own settings for the following
+    1. jdbc (DB) username and password
+    1. MasterEncryptionKey password 
+    1. SMTP settings
+1. Edit the postgres/fixtures/create_fixtures.sql file
+    1.  DB username and password should match those configued in labkey/config/labkey.xml 
```

Comment:
Maybe something here about the `docker-compose.yml` file also needing to match.

Reply:
the postgres password in docker-compose is for the postgres super user and not the db user.  which I also find odd...

Reply:
Added wording to address this and update the docker-compose.yml